sklearn/utils/class_weight.py

```diff
@@ -78,6 +80,26 @@ def compute_class_weight(class_weight, classes, y):
     return weight
 
 
+def _jitter_transform(true_order, bad_order):
```

Comment:
wow, this got big :p

Reply:
I feel I may have gone a little overboard by trying to have a smaller jitter :)